docs/fundamentals/networking/quic/quic-overview.md

```diff
@@ -86,6 +90,27 @@ Here are some examples of using a package manager to install `libmsquic`:
   sudo yum install libmsquic
   ```
 
+#### Installing `libmsquic` from the Distribution Package Repository
+
+Installing `libmsquic` from distribution package repository is also possible, but currently this is only available for `Alpine`.
+
+##### Examples
+
+Here are some examples of using a package manager to install `libmsquic`:
+
+- **Alpine 3.21 and later**
```

Comment:
Alpine 3.21 is not released yet, is it?

Reply:
Not yet, it will be released in short time.